docker/master/Dockerfile

```diff
@@ -8,7 +10,11 @@ FROM debian:jessie
 RUN groupadd -r freeswitch --gid=999 && useradd -r -g freeswitch --uid=999 freeswitch
 
 # grab gosu for easy step-down from root
-RUN gpg --keyserver ha.pool.sks-keyservers.net --recv-keys B42F6819007F00F88E364FD4036A9C25BF357DD4
+RUN apt-get update && apt-get install -y --no-install-recommends dirmngr gnupg2 \
+    && gpg2 --keyserver hkp://keyserver.ubuntu.com --recv-keys B42F6819007F00F88E364FD4036A9C25BF357DD4 \
+    && gpg2 --keyserver hkp://keyserver.ubuntu.com --recv-keys 655DA1341B5207915210AFE936B4249FA7B0FB03 \
+    && gpg2 --output /usr/share/keyrings/signalwire-freeswitch-repo.gpg --export 655DA1341B5207915210AFE936B4249FA7B0FB03 \
+    && apt-get purge -y --auto-remove gnupg2
```

Comment:
Does this break `gpg --verify /usr/local/bin/gosu.asc` ?

Reply:
if `gpg2 --keyserver hkp://keyserver.ubuntu.com --recv-keys B42F6819007F00F88E364FD4036A9C25BF357DD4` remove, I think it is break, but now I do not see any break

Am I correct?

Reply:
Do you remove gnupg2 before calling gpg verify?

Reply:
I try to remove extra layer, with removing `RUN` please wait

Reply:
@andywolk  I submit new PR #1589